router/navigation-drawer.tsx

```diff
@@ -103,8 +103,8 @@ export const navGroupItems2: NavItem[] = [
         itemID: 'g2i3',
     },
     {
-        subtitle: 'You cant see me, fix InfoListItem',
-        title: 'Alerts',
+        subtitle: 'Click me to clear',
+        title: 'Clear Selection',
         itemID: 'g2i4',
```

Comment:
We probably don't need to keep this in here now.

Reply:
I was just leaving it in there for regression testing of the parent font weight stuff. I can remove it though. Do you want the entire item removed, or just remove the functionality and change the title to something generic?

Reply:
If it were me, I would set it back to what it was before. And then if we want to have an item to test the styles, make a new item as the last item in the drawer body.